### ISSUE TITLE
NullPointer in com/britesnow/snow/web/rest/RestRegistry.java

### ISSUE THREAD

**Issue by Ueland** [BriteSnow/snow#44]
The function "getRefByPath" is missing support for "OPTIONS"-request, which is something Microsoft-WebDAV-MiniRedir (MS Office) is sending out, which triggers a NullPointerException.

**Comment by jeremychone** [comment]
Sorry for the late answer. 

Do you still have this issue? How can I reproduce it without Microsoft?

**Comment by Ueland** [comment]
Yes, it stil exist. I suspect you can reproduce it by sending a OPTIONS request instead of a GET/POST request. ;)
